README.rdoc

```diff
@@ -159,11 +159,11 @@ You can pass in your own key when generating a shortened URL. This should be uni
 You can create expirable URLs.
 Probably, most of the time it would be used with owner:
 
-  Shortener::ShortenedUrl.generate("example.com/page", user, expires_at: 24.hours.since)
+  Shortener::ShortenedUrl.generate("example.com/page", user: user, expires_at: 24.hours.since)
```

Comment:
Looking at code https://github.com/jpmcgrath/shortener/blob/53ca7f3b2d57da3f06c22e61ad6ce694663996cd/app/models/shortener/shortened_url.rb#L65, this should be:
```suggestion
  Shortener::ShortenedUrl.generate("example.com/page", owner: user, expires_at: 24.hours.since)
```